proxy/client.go

```diff
@@ -83,6 +103,12 @@ func DefaultClientCreator(addr, transport, dbDir string) ClientCreator {
 			panic(err)
 		}
 		return NewLocalClientCreator(app)
+	case "e2e_sync":
```

Comment:
Shouldn't this be `e2e_unsync`? Isn't it a bit confusing this way

Reply:
I don't think so. It's a synchronized application that we're connecting to with an unsynchronized client.

The `e2e` app is an unsynchronized application with a synchronized client, by comparison.

Reply:
ah I see. I was looking at this from the point of view of the client not the application